apps/web/src/components/AssessmentStepTracker/AssessmentResults.tsx

```diff
@@ -88,19 +145,72 @@ const AssessmentResult = ({
         data-h2-gap="base(0 x.25)"
         data-h2-padding="base(x.125 0)"
         data-h2-width="base(100%)"
+        {...(isBookmarked && {
+          "data-h2-radius": "base(5px)",
+          "data-h2-background-color": "base(primary.lightest)",
+        })}
       >
-        <BookmarkIcon
-          {...iconStyles}
-          data-h2-color="base(gray)"
-          data-h2-flex-shrink="base(0)"
-        />
+        {isBookmarked ? (
+          <Button
+            mode="icon_only"
+            color="primary"
+            onClick={toggleBookmark}
+            icon={BookmarkIconSolid}
+            aria-label={intl.formatMessage(
+              {
+                defaultMessage:
+                  "Un-bookmark {candidateName} from top of column.",
+                id: "qlZMZx",
+                description:
+                  "Un-bookmark button label for applicant assessment tracking.",
+              },
+              {
+                candidateName: getFullNameLabel(
+                  result.poolCandidate.user.firstName,
+                  result.poolCandidate.user.lastName,
+                  intl,
+                ),
+              },
+            )}
+            {...{
+              "data-h2-height": "base(x.9)",
+              "data-h2-width": "base(x.9)",
+            }}
+          />
+        ) : (
+          <Button
+            mode="icon_only"
+            color="black"
+            onClick={toggleBookmark}
+            icon={BookmarkIconOutline}
+            aria-label={intl.formatMessage(
+              {
+                defaultMessage: "Bookmark {candidateName} to top of column.",
```

Comment:
Isn't the verb here meant to be **pin**?

Reply:
I considered this but aren't they synonyms? And we already use an icon of a bookmark not a pin.

Reply:
I've updated this with [Change some copy to use 'pin' instead of 'bookmark'](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8767/commits/bd5d8656334aaf072525789e776fe3d8e36a5131) but I feels like it loses some consistency since we use "Bookmark" elsewhere.

Reply:
@GCTC-NTGC/designers thoughts?

Reply:
Instructions from designers: no copy changes for now